index.html

```diff
@@ -5,6 +5,7 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
+    <link href="Scripts/main.js">
```

Comment:
@Harrisonking1923 Hey man, this is a little wrong. Could you use a script tag for js please.